webapp/views.py

```diff
@@ -11,3 +19,56 @@ def custom_404(request):
 def custom_500(request):
     t = loader.get_template('error/500.html')
     return HttpResponseServerError(t.render(Context({})))
+
+
+class MarkdownView(TemplateView):
+    template_name = 'includes/base_markdown.html'
```

Comment:
Could you pass this through from `urls.py` instead of setting it directly please?